docs/installation/advanced/integrating-from-source-webpack.md

```diff
@@ -59,21 +47,36 @@ npm install --save \
 	webpack-cli@4
 ```
 
+The list will differ if you want to use TypeScript in your project - additionally, you need to install `ts-loader`.
+
+```bash
+npm install --save ts-loader
+```
+
 ## Webpack configuration
 
 You can now configure webpack. There are a couple of things that you need to take care of when building CKEditor 5:
 
 * Handling CSS files of the CKEditor theme. They are included in the CKEditor 5 sources using `import 'path/to/styles.css'` statements, so you need [proper loaders](https://webpack.js.org/loaders/).
 * Similarly, you need to handle bundling SVG icons, which are also imported directly into the source. For that you need the [`raw-loader`](https://webpack.js.org/loaders/raw-loader/).
+* Optionally, you may need to handle `.ts` files to use TypeScript in your project. There is the [`ts-loader`](https://webpack.js.org/guides/typescript/#loader) for this purpose.
 * Finally, to localize the editor you need to use the [`@ckeditor/ckeditor5-dev-translations`](https://www.npmjs.com/package/@ckeditor/ckeditor5-dev-translations) webpack plugin.
 
 The minimal configuration, assuming that you use the same methods of handling assets as CKEditor 5 builds, will look like this:
```

Comment:
Please add separate headers for JavaScript and TypeScript so they can be easily found or skipped.